nodes/get-object/get-object.js

```diff
@@ -2,6 +2,7 @@ module.exports = function (RED) {
   "use strict";
   const { S3 } = require("@aws-sdk/client-s3");
   const { streamToString } = require("../../common/common");
+  const { streamToStringbase64 } = require("../../common/common");
```

Comment:
So it can be used here. [Reference](https://github.com/digitalnodecom/node-red-contrib-generic-s3/pull/48/files#r1265139870)